### PR TITLE
Prow: Add terraform lintern to openshift/installer

### PR DESCRIPTION
**Pull request by alejovicu**
Add Terraform lintern to openshift/installer. (CORS-744)

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -3174,6 +3174,20 @@ presubmits:
         - --template=/usr/local/e2e-aws
         - --target=e2e-aws
 
+  - name: terraform-lint
+    trigger: "(?m)^/tflint"
+    rerun_command: "/tflint"
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: Convention is to use `/test <test-name>`

File: cluster/ci/config/prow/config.yaml [hunk 2]
```diff
@@ -3174,6 +3174,20 @@ presubmits:
...
+    context: terraform-lint
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: Convention is to use `ci/prow/<name>`

**Comment by wking** [comment]
@alejovicu, this needs a rebase (at least according to GitHub :p).

File: cluster/ci/config/prow/config.yaml [hunk 3]
```diff
@@ -3209,6 +3209,20 @@ presubmits:
         - --template=/usr/local/e2e-aws
         - --target=e2e-aws
 
+  - name: terraform-lint
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: sorry, final nit -- this is usually ci-pull-$org-$repo-$name

**Comment by alejovicu** [comment]
Please **do not merge** until  https://github.com/openshift/installer/pull/109 is merged

**Comment by alejovicu** [comment]
Ready for merge

**Pull request merged**